portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,25 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    ArrayList<String> positiveMessages = new ArrayList<String>();
+    positiveMessages.add("Keep your face to the sunshine and you cannot see a shadow");
+    positiveMessages.add("The only time you fail is when you fall down and stay down");
+    positiveMessages.add("If opportunity doesn't knock, build a door");
+    positiveMessages.add("Live life to the fullest and focus on the positive");
+
+    Gson gson = new Gson();
+    String json = gson.toJson(positiveMessages);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
```

Comment:
Can we make these class constants.

You can try using guava ImmutableList and build

```
private static final List<String> messages =
    ImmutableList.builder()
        .add("message-1")
        .add("message-2")
        .build();
```

or 

```
private static final List <String> messages = Arrays.asList("messsage-1", "message-2",....);
```

```
private static final String message_as_json = new Gson().toJson(messages);
```